snippets/liquid-template-snippets.json

```diff
@@ -589,6 +716,672 @@
     "body": "{% ic %}{::warningtext as=\"hover\"}${1:content}{:/warningtext}{% endic %}",
     "description": "The Silverfin templating language gives the opportunity to add warning text. There are 3 ways to present it: as an inline element, as a block or as a hover element."
   },
+
+  ".size": {
+    "prefix": ".size",
+    "body": ".size",
+    "description": [
+      "Return size of an array or string"
+    ]
+  },
+
+  "Date filter - DD/MM/YYYY format": {
+    "prefix": "date: DD/MM/YYYY",
+    "body": "| date:\"%d/%m/%Y\"",
+    "description": "Standard BE date formatting (DD/MM/YYYY)."
+  },
+  "Date filter - YYYY-MM-DD format": {
+    "prefix": "date: YYYY-MM-DD",
+    "body": "| date:\"%F\"",
+    "description": [
+      "ISO date formatting (DD/MM/YYYY).\n",
```

Comment:
Description should still be updated here as well

Reply:
Updated